### PR TITLE
docs: add rosbag2 pre process

### PR DESCRIPTION
**Pull request by hayato-m126**
## Description

Added section on rosbag2 pre-processing

## How to review

please review pre-process command

### test data

no need

### test command

```bash

```

### CODE REVIEW

File: docs/tools_overview.md [hunk 1]
```diff
@@ -15,6 +15,30 @@ source ${ROS_WORKSPACE_WITH_CUSTOM_MESSAGES}/install/setup.bash
 poetry shell
 ```
 
+## rosbag2 pre-process
+
+The bag of t4_dataset must contain the TOPIC below.
+[topics included in t4_dataset bag](t4_format_3d_detailed#input-bag)
```
Comment (miursh) [inline docs/tools_overview.md:21]:
```suggestion
[topics included in t4_dataset bag](t4_format_3d_detailed.md#input-bag)
```

File: docs/tools_overview.md [hunk 2]
```diff
@@ -15,6 +15,30 @@ source ${ROS_WORKSPACE_WITH_CUSTOM_MESSAGES}/install/setup.bash
...
+output: rosbag2
+
+```bash
+# use autoware
+# terminal 1
+source ${AUTOWARE_WORKSPACE}/install/setup.bash
+ros2 launch autoware_launch logging_simulator.launch.xml map_path:=${MAP_PATH} vehicle_model:=${VEHICLE_MODEL} vehicle_id:=${VEHICLE_ID} sensor_model:=${SENSOR_MODEL} vehicle:=true sensing:=true perception:=false planning:=false control:=false map:=true localization:=true rviz:=true
+# terminal 2
+# PLAY_RATE example is 0.2. Higher rates may be possible depending on computer processing power. remap recorded /tf
+ros2 bag play ${BAG_BEFORE_PROCESSING} --remap /tf:=/unused_tf --rate ${PLAY_RATE} --clock 200
```
Comment (miursh) [inline docs/tools_overview.md:37]: If the input rosbag contains a correctly positioned /tf topic, enabling localization:=true and remapping /tf:=/unused_tf  are not necessary.
Therefore, could you please add a description stating that if /tf topic with map to base_link is already recorded, the option should be like "localization:=false" and  no "--remap /tf:=/unused_tf".

Comment (hayato-m126) [reply docs/tools_overview.md:51]: I updated the description with commit id b94c28ff586bf564f1914d3ef82b72e1fb87c0f5

**Review by miursh — APPROVED** [review]
LGTM

**Pull request merged**